packages/lesswrong/components/editor/PostCollaborationEditor.tsx

```diff
@@ -108,7 +111,7 @@ const PostCollaborationEditor = ({ classes }: {
           accessLevel={post.myEditorAccess as CollaborativeEditingAccessLevel}
           document={post}
         />
-        <Components.PostVersionHistoryButton
+        <VersionHistoryButton
```

Comment:
Yeah, the collab editor would be much trickier.  In that editor, after it's initialized, the state of the document is managed by ckEditor's remote collaboration session.  Separately, it'd be very difficult to safely "Load into editor" there, since after soft-loading, ckEditor would send us an event via webhook which would trigger an autosave, causing that revision to get saved as the latest revision again anyways.